proposals/lip-0057.md

```diff
@@ -856,7 +856,13 @@ Transactions executing this command have:
 
 ##### Parameters
 
-The `params` property of unlock transactions is empty.
+```java
+unlockTransactionSchema = {
+    "type": "object",
+    "required": [],
+    "properties": {}
+}
+```
```

Comment:
The schema validation in the `verify` hook is missing.

Reply:
added